### PR TITLE
fix/31804/chat-fixes

### PR DESCRIPTION
**Pull request by massimez**

### CODE REVIEW

File: components/app/pages/messages_id/messagesList.vue [hunk 1]
```diff
@@ -92,18 +92,34 @@
                     alt=""
                     @click="selFile($event, message.medias, file.url)"
                   >
+                  <div
+                    v-else-if="file.type === 'video'"
```
Comment (matykoks) [inline components/app/pages/messages_id/messagesList.vue:96]: make enum for it
compare with object of keys your_obj.VIDEO
same changes need for type of users
write it in one row

File: pages/messages/_id/index.vue [hunk 2]
```diff
@@ -25,12 +25,17 @@
                 {{ currChat && currChat.questChat && currChat.questChat.quest.title }}
               </div>
               <div
-                v-if="currChat.type === ChatType.PRIVATE"
+                v-if="currChat.type === ChatType.PRIVATE && privateCorrespondentMember.type === 'User'"
                 class="chat-container__quest-link"
-                @click="$router.push(`${$options.Path.PROFILE}/${privateCorrespondentMember.user.id}`)"
+                @click="$router.push(`${$options.Path.PROFILE}/${privateCorrespondentMember.userId}`)"
```
Comment (matykoks) [inline pages/messages/_id/index.vue:30]: i am not sure, but i think you changed it every pr?

File: pages/messages/_id/index.vue [hunk 3]
```diff
@@ -131,6 +136,16 @@
                 :title="file.file.name"
                 @click="openFile"
               >
+                <video
+                  v-if="file.type === 'video'"
```
Comment (matykoks) [inline pages/messages/_id/index.vue:140]: enum

Comment (matykoks) [reply pages/messages/_id/index.vue:140]: + 151 row

File: pages/messages/_id/index.vue [hunk 4]
```diff
@@ -199,6 +216,7 @@ export default {
       const {
         isClosedQuestChat, isGroupChat, amIOwner, isPrivateChat,
       } = this;
+      if (this.chatId === 'starred') return false;
```
Comment (matykoks) [inline pages/messages/_id/index.vue:219]: can you take it from ChatType?

File: utils/enums.js [hunk 5]
```diff
@@ -363,3 +363,13 @@ export const EntityType = {
   QUEST: 'Quest',
   DISCUSSION_COMMENT: 'DiscussionComment',
 };
+
+export const UserRoles = {
```
Comment (matykoks) [inline utils/enums.js:367]: look at UserRole on 159 row

File: pages/messages/_id/index.vue (hunk 4 above)
Comment (massimez) [reply pages/messages/_id/index.vue:219]: No, isnt the same

**Pull request merged**